### ISSUE TITLE
assign background value, "show raw image(s)"

### ISSUE THREAD

**Issue by martinschorb** [mobie/mobie-viewer-fiji#733]
When I try to "show raw image" of a source that is not a complete rectangle, the values outside of existing data get padded with `0`.
Could you add an option to define the pad value?

**Comment by tischi** [comment]
> that is not a complete rectangle

Could you let me know how to reproduce this? 
The "raw data" by definition should be a complete rectangle, because it really should just give you what is stored on disk.

**Comment by martinschorb** [comment]
the data is in here:
`/g/schwab/Beckwith_MSB/Microscopy_data/SBEM_data/220325_MSB25_sample2_HL60_lifeact_liveCellCLEM/aligned/20220503-1326`

It is sparse in a sense that there are only active tiles here and there (changing during the run through `z`). I'd like the outside of the tiles to be filled with the value roughly matching the empty resin instead of being zero.

I can obviously script it to replace all zeros in the output volume with something else, but having this handled right upon export would be more elegant.

**Comment by tischi** [comment]
> I can obviously script it to replace all zeros in the output volume with something else, but having this handled right upon export would be more elegant.

I think in OME-Zarr you can specify the Fill Value in one of the JSON.
What file format do you use here?

**Comment by martinschorb** [comment]
so far N5, it's produced by Render.

**Comment by tischi** [comment]
does that have metadata for the fill value?
@bogovicj

**Comment by constantinpape** [comment]
No, not supported in n5, see https://github.com/saalfeldlab/n5#file-system-specification.
